astor/node_util.py

```diff
@@ -132,6 +132,7 @@ class ExplicitNodeVisitor(ast.NodeVisitor):
 
     """
 
+    @staticmethod
     def abort_visit(node):  # XXX: self?
```

Comment:
Uhhh.  Yeah.  (Mumbles to himself about why he did something based on a comment then didn't remove the comment.)